UefiPayloadPkg/Include/Guid/UniversalPayloadSerialPortDeviceParentInfo.h

```diff
@@ -0,0 +1,25 @@
+/** @file
+  Universal Payload serial port parent device information definitions.
+
+Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef UNIVERSAL_PAYLOAD_SERIAL_PORT_PARENT_DEVICE_INFO_
+#define UNIVERSAL_PAYLOAD_SERIAL_PORT_PARENT_DEVICE_INFO_
+
+extern GUID  gUniversalPayloadSerialPortParentDeviceInfoGuid;
+
+#pragma pack(1)
+typedef struct {
+  UINT32     Revision;
+  BOOLEAN    IsUnderPci;

```

Comment:
Even the parent device of serial port device is ISA bus controller, it may be still PCI device, so let's rename this field to IsIsaCompatible to be clear. Please also add comments to this field.

Example description of IsIsaCompatible field:
TRUE: the serial port device is under an ISA compatible bus, which means the parent device is ISA/LPC/eSPI bus controller.
FALSE: the serial port device is native PCI device under PCI bridge.

Reply:
Todo.

Reply:
resolved